### DISCUSSION TITLE
identifier "GRIB" not defined in this scope!

### DISCUSSION THREAD

**Discussion by sherrieF** [dtcenter/METplus#1239] [File I/O] [answered]
Good Morning,
          I am getting an erorr of 
OUTPUT:
104 DEBUG 1: Default Config File: /contrib/met/10.0.0/share/met/config/GridStatConfig_default
105 DEBUG 1: User Config File: /contrib/METplus/METplus-4.0.0/parm/met_config/GridStatConfig_wrapped
106 ERROR  :
107 ERROR  : do_assign_id() -> identifier "GRIB" not defined in this scope!
108 ERROR  :
109 11/04 14:57:08.463 metplus (command_builder.py:255) ERROR: (command_builder.py:1309) MET command returned a non-zero return code:/cont    rib/met/10.0.0/bin/grid_stat -v 2 /scratch2/BMC/gsienkf/Sherrie.Fredrick/C96_replay_winter/2015120400/GFSPRS.GrbF00.grib2 /scratch2/BM    C/gsienkf/Sherrie.Fredrick/era5/erai_2015120400.grib /contrib/METplus/METplus-4.0.0/parm/met_config/GridStatConfig_wrapped -outdir /sc    ratch2/BMC/gsienkf/Sherrie.Fredrick/C96_replay_winter/metout/2015120400

I am working on Hera.  My config file is in /home/Sherrie.Fredrick/METplus/gridstat on Hera.
many thanks for your help.
Sherrie

**Comment by sherrieF** [comment]
I was able fix this problem.  Thanks

**Reply by jprestop** [reply]
Hi Sherrie.  I'm glad you were able to figure out what the problem was.   Would you consider editing your comment to include the solution that worked for you in order to help other users who may have encountered this same problem?  Was it the case that the the file_type value (set by FCST_GRID_STAT_FILE_TYPE and OBS_GRID_STAT_FILE_TYPE) had to be GRIB1 or GRIB2 but not just GRIB?

**Comment by sherrieF** [comment, marked as answer]
I changed the OBS_GRID_STAT_FILE_TYPE to GRIB1

**Reply by jprestop** [reply]
Thank you, Sherrie.  At this point I'm going to lock this conversation to prevent future posts. Our team has decided that once discussions have been answered, and the answer has been confirmed, we'll lock them. We want to encourage users to ask new questions in new discussions rather than posting to old ones, just like you did here - thank you! Hopefully that'll make the questions/answers easier for other users to follow. So if/when more issues/questions arise, please feel free to start a new discussion.
